creator/ingest_runs/models.py

```diff
@@ -77,6 +85,35 @@ class Meta:
         help_text="The user who submitted this ingest run",
     )
 
+    @transition(field=state, source="waiting", target="started")
+    def start(self):
+        """
+        Begin running the ingest process.
+        """
+        return
+
+    @transition(field=state, source="started", target="complete")
```

Comment:
Sorry plz don't kill me... small request. I think we should change it to  `target=completed` since all of the other states are in past tense.